### PR TITLE
weaver 0.10.0 (new formula)

### PR DESCRIPTION
**Pull request by trupin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by trupin** [comment]
I just bumped the version to 0.9.10 and rebased the branch on master.

Is there anything more I should do to get this merged?

File: Formula/weaver.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def install
+    # libxml2 has to be included in ISYSTEM_PATH for building one of
+    # dependencies. It didn't happen automatically before Xcode 9.3
+    # so homebrew patched environment variable to get it work.
+    # But since Xcode 9.3 includes it already, the build will fail
+    # because of redefinition of libxml2 module.
+    # It's a bug of homebrew but before it's fixed, it's easier
+    # to provide in-place workaround for now.
+    # Please remove this once homebrew is patched.
+
```
Comment (DomT4) [inline Formula/weaver.rb]: Can we trim this empty line here and in `test do`? Thanks!

Comment (trupin) [reply Formula/weaver.rb]: Done in 8077f41f4b

**Comment by DomT4** [comment]
Looks like we're stuck with the `ISYSTEM_PATH` issue for now, so I guess we'll roll with it in the wild and see how it goes.

Merged in https://github.com/Homebrew/homebrew-core/commit/2baaf853691e4a287eae11d2a82cea5dad1be61b. Thank you for your contribution to Homebrew @trupin; we appreciate it! :smiley_cat:

**Comment by trupin** [comment]
Great! @DomT4, thank you so much for merging this.

**Pull request closed without merging**